experiments/README.md

```diff
@@ -8,37 +8,19 @@ macOS to install some GNU-compatible binaries before all experiments scripts wil
 brew install coreutils gnu-getopt parallel
 ```
 
-## Setup
+## Scripts
 
-Phase 1 requires [AWS CLI](https://aws.amazon.com/cli/) because it downloads data from AWS S3.
+### Phase 1: Generate expert demonstrations from models.
 
-### Phase 1: Download RL (PPO2) expert policies and AIRL/GAIL reward models.
+Run `experiments/rollouts_from_policies.sh`. (Rollouts saved in `output/train_experts/`).
+Demonstrations are used in Phase 2 for imitation learning.
 
-Use `experiments/download_models.sh`. (Downloads to `data/{expert,reward}_models/`).
-Expert policies are used in Phase 2 to generate demonstrations.
-Reward models are used in Phase 4 for transfer learning.
-
-Want to use other policies or reward models for Phase 2 or 4? 
-  * New policies can be trained using `experiments/train_experts.sh`. 
-  * New reward models are generated in Phase 3 `experiments/imit_benchmark.sh`.
-  * For both scripts, you can enter the optional commands suggested at the end of the script to upload new files to S3 (will need write access to our S3 bucket). Or, you can manually patch `data/{expert,reward}_models` using the script's output files.
-
-
-### Phase 2: Generate expert demonstrations from models.
-
-Run `experiments/rollouts_from_policies.sh`. (Rollouts saved in `data/expert_models/`).
-Demonstrations are used in Phase 3 for imitation learning.
-
-### Phase 3: Train imitation learning.
+### Phase 2: Train imitation learning.
 
 Run `experiments/imit_benchmark.sh --run_name RUN_NAME`. To choose AIRL or GAIL, add the `--airl` and `--gail` flags (default is GAIL).
 
 To analyze these results, run `python -m imitation.scripts.analyze with run_name=RUN_NAME`. Analysis can be run even while training is midway (will only show completed imitation learner's results). [Example output.](https://gist.github.com/shwang/4049cd4fb5cab72f2eeb7f3d15a7ab47)
 
-### Phase 4: Transfer learning.
```

Comment:
Why did you delete this? If it was because `experiments/transfer_benchmark.sh` doesn't exist I think it just got renamed to `experiments/transfer_learn_benchmark.sh`.

Reply:
I think maybe I deleted it because currently stuff doesn't save to `data/reward_models`. I guess I can reinstate it and leave fixing it for a general "make the experiment scripts work" PR.

Reply:
Have reinstated it.

Reply:
Ah yeah that is a problem but agree probably best to punt it to another issue. We should really overhaul the whole `experiments/` directory -- I expect @taufeeque9 will end up doing this as part of his benchmarking work.